cores/esp8266/Esp.h

```diff
@@ -81,6 +81,8 @@ typedef enum {
      FM_UNKNOWN = 0xff
 } FlashMode_t;
 
+typedef void (*CrashCallback)( struct rst_info *, uint32_t, uint32_t );
```

Comment:
I couldn't find a suitable header to put this typedef in to avoid duplicated the sig in other files, and I was hesitant to create a new header just for this one thing.